sdk/python/setup.py

```diff
@@ -117,16 +117,23 @@
     "pre-commit",
     "assertpy==1.1",
     "pip-tools",
-] + GCP_REQUIRED + REDIS_REQUIRED + AWS_REQUIRED
+    *GCP_REQUIRED,
+    *REDIS_REQUIRED,
+    *AWS_REQUIRED,
+]
 
-DEV_REQUIRED = ["mypy-protobuf==1.*", "grpcio-testing==1.*"] + CI_REQUIRED
+DEV_REQUIRED = [
+    "mypy-protobuf==1.*",
+    "grpcio-testing==1.*",
+    *CI_REQUIRED,
+]
 
 # Get git repo root directory
 repo_root = str(pathlib.Path(__file__).resolve().parent.parent.parent)
 
 # README file from Feast repo root directory
 README_FILE = os.path.join(repo_root, "README.md")
-with open(README_FILE, "r") as f:
+with open(README_FILE, "r", encoding="utf8") as f:
```

Comment:
Nit: I believe the correct name is `utf-8`

Reply:
`utf-8` was my first option before I look for references and usage in other source files and they was using `utf8`:

https://github.com/feast-dev/feast/blob/7538f7e643765b5abcf004e3cea2918928c8f0dc/sdk/python/feast/infra/key_encoding_utils.py#L34

`utf8` is valid according the aliases column in the [Standard Encodings](https://docs.python.org/3/library/codecs.html#standard-encodings) table.

Do you think it's better to switch to `utf-8`?

Reply:
I don't have a strong preference. If that's a standardized alias, let's keep it.